src/Tempest/Database/Migrations/MigrationManager.php

```diff
@@ -27,21 +28,21 @@ public function up(): void
         try {
             $existingMigrations = Migration::all();
         } catch (PDOException) {
-            $this->executeUp(new CreateMigrationsTable());
+            $this->executeUp(new CreateMigrationsTable($this->databaseConfig->driver()));
 
             $existingMigrations = Migration::all();
         }
 
         $existingMigrations = array_map(
-            fn (Migration $migration) => $migration->name,
+            static fn (Migration $migration) => $migration->name,
             $existingMigrations,
         );
 
-        foreach ($this->databaseConfig->migrations as $migrationClassName) {
+        foreach ($this->databaseConfig->getMigrations() as $migrationClassName) {
```

Comment:
normal sort, would otherwise cause errors when dealing with foreign keys #297 
```suggestion
        $migrations = $this->databaseConfig->getMigrations();
        ksort($migrations);
        foreach ($migrations as $migrationClassName) {
```